doc/src/manual/methods.md

```diff
@@ -189,7 +189,7 @@ Whoa there, Nelly.
 ```
 
 This catch-all is less specific than any other possible method definition for a pair of parameter
-values, so it is only be called on pairs of arguments to which no other method definition applies.
+values, so it is only to be called on pairs of arguments to which no other method definition applies.
```

Comment:
Idk, I read "it is only to be called on pairs...." as "you should only call this method on pairs of..." rather than "this method will only be invoked if no other method is more specific". But not a native speaker so `¯\_(ツ)_/¯`